### PR TITLE
feat(compiler): support for ConfigRollout

### PR DESCRIPTION
**Pull request by smintz**

### CODE REVIEW

File: compiler/lib/compiler.go [hunk 1]
```diff
@@ -261,7 +266,8 @@ func (c *Compiler) load(filename string) (*config, error) {
 
 func (c *Compiler) GetLoader() *starlarkLoader {
 	modules := getModules()
-	modules["remote_repo"] = starlark.NewBuiltin("remote_repo", c.ModuleService.Add)
+	modules["ConfigRollout"] = starlark.NewBuiltin("ConfigRollout", newConfigRollout)
```
Comment (avivl) [inline compiler/lib/compiler.go:269]: Can we have the "ConfigRollout"  "RolloutStage" ets defined as consts?

File: compiler/lib/compiler.go [hunk 2]
```diff
@@ -271,3 +277,41 @@ func (c *Compiler) GetLoader() *starlarkLoader {
 		moduleService: c.ModuleService,
 	}
 }
+
+func newConfigRollout(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var config starlark.Value
+
+	starlark.UnpackArgs(fn.Name(), args, kwargs, "value", &config)
```
Comment (avivl) [inline compiler/lib/compiler.go:284]: check for error

**Review by avivl — CHANGES_REQUESTED** [review]

File: compiler/lib/compiler.go (hunk 1 above)
Comment (smintz) [reply compiler/lib/compiler.go]: done.

File: compiler/lib/compiler.go (hunk 2 above)
Comment (smintz) [reply compiler/lib/compiler.go]: I've added some comments in regard to why the error check was skipped and added a different check functionality.

**Review by avivl — APPROVED** [review]

**Pull request merged**